server/index.ts

```diff
@@ -0,0 +1,58 @@
+import express, { Request, Response } from "express";
+import dotenv from "dotenv";
+import cors from "cors";
+import pool from "./pg";
+
+// configures dotenv to work in your application
+dotenv.config();
+
+const app = express();
+
+const PORT = process.env.PORT;
+
+app.use(cors())
+
+app.get("/", async (request: Request, response: Response) => {
+  const { asset, age } = request.query
+  const page = Number(request.query.page ?? 0)
+  const pageSize = Number(request.query.pageSize ?? 10)
+
+  const where = ["TRUE"], args = []
+
+  if (asset !== undefined) {
+    where.push("(src_asset = $ OR dest_asset = $)")
+    args.push(asset)
```

Comment:
Sanitize inputs - check and make sure they belong to a list (or at least match address regex)

Reply:
Done